pkg/camo/proxy.go

```diff
@@ -221,10 +221,17 @@ func (p *Proxy) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 
 	switch resp.StatusCode {
 	case 200:
-		// check content type
+		contentType := resp.Header.Get("Content-Type")
+
+		if contentType == "" {
```

Comment:
Am I correct in my assumption that this is an optimization?

The regex match /should/ fail (non-match) for empty content types, but not having to do the regex in cases where content-type is empty seems like a valid optimization case for sure. Just wanted to make sure the reasoning matched my expectation.

Reply:
Not an optimisation but ability to differentiate an unsupported content type and a missing one. These are two different scenarios and while you could lump them together, you probably want to have insight into each case instead of treating both the same way.

Apologies for not being clear enough about that in my description.

Reply:
Ah. That makes sense. Treating them differently here isn't really leaking anything of value either. 👍